libs/api/domains/license-service/src/lib/graphql/main.resolver.ts

```diff
@@ -135,6 +139,7 @@ export class MainResolver {
   }
 
   @Mutation(() => GenericPkPassVerification)
+  @Audit()
```

Comment:
Might be good to add some ids or meta to these audit records. Ie which licenses and/or license types are we generating pkpass for